frontend/packages/erd-core/src/components/ERDRenderer/Toolbar/MobileToolbar.module.css

```diff
@@ -1,176 +0,0 @@
-@keyframes collapse {
-  from {
-    width: 170px; /* TODO: 157px */
-    height: 245px;
-  }
-  to {
-    width: 40px;
-    height: 40px;
-  }
-}
```

Comment:
Managing three states (closed, opened(1), opened(2)) with keyframes was difficult, so I switched to using `transition: width ... height ...` instead.